govcd/lb_test.go

```diff
@@ -263,12 +263,14 @@ func checkLb(queryUrl string, expectedResponses []string, maxRetryTimeout int) e
 		iterations = maxRetryTimeout / 5
 	}
 
+	httpClient := &http.Client{Timeout: 5 * time.Second}
```

Comment:
Could you add a comment explaining how this timeout affects the expected duration of the test?

I am talking about the estimate given with `Waiting for the virtual server to accept responses (%s interval x %d iterations)"`

Reply:
Yep. This simple loop was actually quite unstable in guessing timing. I put in a `time.Ticker` so that it does not lie and actually work as the time was set.